CITATION.cff

```diff
@@ -0,0 +1,31 @@
+# This CITATION.cff file was generated with cffinit.
```

Comment:
Did Ian's PR get merged? If so, maybe we should add him?

Reply:
https://github.com/choderalab/feflow/pull/44 not yet, but definitely something to keep in mind once that's done!